test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -59,6 +60,12 @@ def collection_name(v, error_code=None):
     return v
 
 
+def deprecation_versions():
+    ''' Create a list of valid version for deprecation entries, current+4 '''
+    current = float('.'.join(__version__.split('.')[0:2]))
+    return Any(*[str(current + x / 100) for x in range(0, 5)])
```

Comment:
This avoids using floats/division and uses `"""` for docstrings (follows PEP 8 style).

```suggestion
def deprecation_versions():
    """Create a list of valid version for deprecation entries, current+4"""
    major, minor = [int(version) for version in __version__.split('.')[0:2]]
    return Any(*['{0}.{1}'.format(major, minor + increment) for increment in range(0, 5)])
```

The use of floats/division can yield incorrect results:

```
'2.12' -> 2.12 -> ['2.12', '2.13', '2.14', '2.15', '2.16']
'2.13' -> ['2.13', '2.1399999999999997', '2.15', '2.1599999999999997', '2.17']
```